### ISSUE TITLE
weird behavior with multiple screens setup

### ISSUE THREAD

**Issue by FSLMobile** [symless/synergy#5489]
My setup is 
Windows 10 (surface book with 3 screens) (named PC0 below)
OSx 10.10 (Macbook pro with 2 screens) (named PC1)

I can switch only from main screen to main screen:

PC0-0 PC0-1 PC0-2 / PC1-0 PC1-1 -> Not working (mouse never go from PC0-2 to PC1-0).
PC1-1 PC1-0 / PC0-0 PC0-1 PC0-2 -> Working (mouse is going fine from PC1-0 to PC0-0).

As long as the main screens are along, this is working. As soon as they are not, the mouse is stucked on the latest screen (PC0-2 in my case).

In both case, setup is working fine and client and server discover and connect without trouble.

It was working fine with previous released version.

**Comment by XinyuHou** [comment]
@FSLMobile 
What version of Synergy are you using?

Could you send us some logging by using gist?
https://gist.github.com/

**Comment by XinyuHou** [comment]
Closed because of insufficient information.

**Comment by FSLMobile** [comment]
Sorry, I was in vacation.

The last version I tested is the last one (1.83 rc1), The problem is still there.

Attached is log from the program folder.

The setup is this one :

In Synergy configuration, the system is defined as
Surface / MacPro
[cid:image001.png@01D1FEBE.E7D1C220]

On each computer, I have 2 or 3 screens (2 in current setup).

Surface screen1 / Surface screen2 / MacPro Screen2 /MacPro Screen1

Surface is Server

In this config, the mouse get stuck between Surface screen2 and cannot go to MacPro Screen2

If I just change the synergy config, moving the Macfred computer to the left:
MacPro/Surface
[cid:image002.png@01D1FEBE.E7D1C220]
The PC remains server

The config is working fine.

Let me know if there is anything else I can do to help.

Cheers,

Fred


De : Jerry (Xinyu Hou) [mailto:notifications@github.com]
Envoyé : mardi 2 août 2016 11:28
À : symless/synergy <synergy@noreply.github.com>
Cc : (FSL) Frederic Tachet <frederic.tachet@fslmobile.fr>; Mention <mention@noreply.github.com>
Objet : Re: [symless/synergy] weird behavior with multiple screens setup (#5489)


@FSLMobile<https://github.com/FSLMobile>
What version of Synergy are you using?

Could you send us some logging by using gist?
https://gist.github.com/

**Comment by XinyuHou** [comment]
https://github.com/symless/synergy/files/442682/synergyd.txt

**Comment by XinyuHou** [comment]
@FSLMobile 
Could you turn up the log level to Debug1 then send us some more logging please?

**Comment by FSLMobile** [comment]
Hi,

Here are the logs file taken on the server.

Can the screen resolution of the Surface be a part of the problem (3000x2000)?

Thanks,

Fred

[synergy not working.txt](https://github.com/symless/synergy/files/445351/synergy.not.working.txt)
[synergy working.txt](https://github.com/symless/synergy/files/445352/synergy.working.txt)
![working](https://cloud.githubusercontent.com/assets/3227495/18096995/6c2ec1ac-6edc-11e6-95f5-4d8bc1299e39.png)
![not working](https://cloud.githubusercontent.com/assets/3227495/18096994/6c23d800-6edc-11e6-95ec-989a5048c2ef.png)

**Comment by FSLMobile** [comment]
Hi, 

As I also had some problem with mouse cursor, I tested this : https://github.com/symless/synergy/issues/4041, And since then, everything is working fine.

**Comment by nlyan** [comment]
@FSLMobile can you elaborate on what you mean by testing #4041 ? The fix for this was merged. Did you disable DPI scaling to fix this issue?

**Comment by FSLMobile** [comment]
Yes, this is exactly what I’ve done. Best workaround of 4041 solved it. I know fix for 4041 was merged long time ago, but it seems some hdpi problem are still there.

Fred

De : Andrew Nelless [mailto:notifications@github.com]
Envoyé : vendredi 9 septembre 2016 13:03
À : symless/synergy <synergy@noreply.github.com>
Cc : (FSL) Frederic Tachet <frederic.tachet@fslmobile.fr>; Mention <mention@noreply.github.com>
Objet : Re: [symless/synergy] weird behavior with multiple screens setup (#5489)


@FSLMobile<https://github.com/FSLMobile> can you elaborate on what you mean by testing #4041<https://github.com/symless/synergy/issues/4041> ? The fix for this was merged. Did you disable DPI scaling to fix this issue?

**Comment by XinyuHou** [comment]
@FSLMobile 
Could you test this nightly please? This nightly includes high DPI fixes for Windows server.
http://symless.com/nightly?filter=synergy-v1.8.6-rc1-eb56683-Windows

**Comment by FSLMobile** [comment]
Forgot to tell you after my tests.

Everything seems to work fine now.

Thanks,

Fred

De : Jerry (Xinyu Hou) [mailto:notifications@github.com]
Envoyé : mardi 8 novembre 2016 12:50
À : symless/synergy <synergy@noreply.github.com>
Cc : (FSL) Frederic Tachet <frederic.tachet@fslmobile.fr>; Mention <mention@noreply.github.com>
Objet : Re: [symless/synergy] weird behavior with multiple screens setup (#5489)


@FSLMobile<https://github.com/FSLMobile>
Could you test this nightly please? This nightly includes high DPI fixes for Windows server.
http://symless.com/nightly?filter=synergy-v1.8.6-rc1-eb56683-Windows
